pkg/conditions/setter.go

```diff
@@ -92,6 +92,11 @@ func MarkFalse(to Setter, t string, reason string, messageFormat string, message
 	to.SetConditions(c4g(to).MarkFalse(t, reason, messageFormat, messageArgs...))
 }
 
+// MarkFalseErr sets Status=False for the condition with the given type.
+func MarkFalseErr(to Setter, t string, reason string, err error) {
```

Comment:
Could just name it `MarkError` ?

Reply:
Agreed. I don't think we will ever have `MarkTrueError` or `MarkUnknownError`.